### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by mregen**
# Change type 

- [ ] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [x] The change requires a change to the documentation
- [x] I have updated the documentation accordingly

# Description and Motivation 

...

### CODE REVIEW

File: docs/opcvault-services-overview.md [hunk 1]
```diff
@@ -103,15 +106,15 @@ For example the application Uri in the database must match the application Uri i
...
+- Another option is to delete just a single certificate request if only a key is renewed or compromised.
 - Now compromised Approved and Accepted certificate requests are marked as deleted.
 - A manager may regularly execute a renewal of the Issuer CA CRL. At the renewal time all the deleted Certificate Requests are revoked and the certificate serial numbers are added to the CRL revocation list. Revoked certificate requests are marked as revoked.
 - In urgent events, single certificate requests can be revoked, too.
 - Finally the updated CRLs are available for distribution to the participating OPC UA clients and servers.
 
 See swagger documentation of the OPC Vault Microservice for more information on the API.
 
-## OPC Vault Module
+## OPC Vault Edge Module
```
Comment (marcschier) [inline docs/opcvault-services-overview.md]: "IoT Edge"

Comment (mregen) [reply docs/opcvault-services-overview.md]: lets have marketing decide this ...

File: README.md [hunk 2]
```diff
@@ -1,91 +1,79 @@
...
-
-The implementation follows the GDS Certificate Management Services as described in the OPC UA specification Part 12.
+[![Build status](https://msazure.visualstudio.com/One/_apis/build/status/Custom/Azure_IOT/Industrial/Components/ci-azure-iiot-opc-vault-service)](https://msazure.visualstudio.com/One/_build/latest?definitionId=44197)
 
-The CA certificates are stored in a HSM backed Azure Key Vault, which is also used to sign issued certificates. 
+#### A detailed overview of the OPC UA Certificate Management Service is [here](docs/opcvault-services-overview.md).
 
-A web management application front end and a local OPC UA GDS server allow for easy connection to the services secured by Azure AD.
+The certificate management service for OPC UA facilitates the **OPC Vault Microservice** to implement the CA certificate cloud service for OPC UA devices based on Azure Key Vault and Azure Cosmos DB, a ASP.Net Core **Sample Certificate Management Web Application** front end and a **OPC Vault Edge Module** to implement a OPC UA GDS server based on the open source OPC UA .Net Standard Nuget packages.
```
[the remark this answers is not in the document]
Comment (mregen) [reply README.md]: yes indeed, shortened

File: docs/opcvault-services-overview.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
-### Azure Industrial IoT OPC Unified Architecture (OPC UA) Certificate Management Service
+### Azure Industrial IoT Service
```
[the remark this answers is not in the document]
Comment (mregen) [reply docs/opcvault-services-overview.md]: yup

**Pull request merged**